OpenOversight/app/templates/tutorial.html

```diff
@@ -25,7 +25,8 @@ <h1>Volunteer Tutorial</h1>
       <div class="row">
         <div class="col-sm-6 col-md-6">
```

Comment:
It's small and I know this is tech-debt, but there are a handful of HTML elements on this page that have the `col-sm-6 col-md-6` classes. Could you change them to just `col-sm-6`? It's redundant to have both.